src/i18n/strings/en_EN.json

```diff
@@ -2644,6 +2646,10 @@
             "device_not_verified_description": "You need to verify this device in order to view your encryption settings.",
             "device_not_verified_title": "Device not verified",
             "dialog_title": "<strong>Settings:</strong> Encryption",
+            "identity_needs_reset": {
+                "description": "You have to reset your cryptographic identity in order to ensure access to your message history",
```

Comment:
Seems to be a duplicate of [`encryption|identity_needs_reset_description`](https://github.com/element-hq/element-web/pull/31367/files#diff-20527b4ae184e85263f5a70fc00d1cb022b054ea126e404332da46983247e5bcR974), perhaps you can use this key instead (or vice versa)?

Reply:
done